readme.md

```diff
@@ -195,6 +216,7 @@ Find Contacts by given options. If you don't set a field to true, it is not incl
         - familyName (boolean) - *default: true*
         - phoneNumbers (boolean)
         - emails (boolean)
+    - baseCountryCode : *default: null (3166-1 alpha-2 countrycode)*    
```

Comment:
Please align correctly and stick to the description of the other values: (type) - *default: value*

Reply:
Currently `baseCountryCode` is not a part of `fields` option, and is on the same level as `fields`:
```js
var options = {
   fields: {
       phoneNumbers: true
   },
   baseCountryCode : 'GB'
};
```